### ISSUE TITLE
BrowserModule has already been loaded. Using BrowserAnimationsModule.

### ISSUE THREAD

**Issue by infacto** [angular/angular#44385]
### Which @angular/* package(s) are the source of the bug?

animations

### Is this a regression?

No

### Description

I have imported `BrowserAnimationsModule` into root app module. But page from another module wants: 

> Found the synthetic property @tabAnimation. Please include either "BrowserAnimationsModule" or "NoopAnimationsModule" in your application.

But adding this module to this causes the error: 

> BrowserModule has already been loaded. If you need access to common directives such as NgIf and NgFor from a lazy loaded module, import CommonModule instead.

Why this error? Angular wants the `BrowserAnimationsModule`, but when adding `BrowserAnimationsModule` it claims that it is already imported? Or what's the problem here?

app-routing,module.ts

```ts
const routes: Routes = [
  {
    path: '',
    component: RootComponent,
    children: [{ path: '', component: LandingPage }],
  },
  {
    path: 'dashboard',
    loadChildren: async () => import('./modules/dashboard.module').then((m) => m.DashboardModule),
  },
  { path: '', redirectTo: 'dashboard', pathMatch: 'full' },
];

@NgModule({
  imports: [RouterModule.forRoot(routes, { enableTracing: false })],
  exports: [RouterModule],
})
export class AppRoutingModule {}
```

The `BrowserAnimationsModule` is imported in AppModule for e.g. `LandingPage`. But the Dashboard wants it too in `DashboardModule`. Adding this causes the error (see above). 

### NG Version

```true
Angular CLI: 12.1.2
Node: 14.15.1
Package Manager: npm 6.14.8
OS: win32 x64

Angular: 12.1.2
... animations, cdk, cli, common, compiler, compiler-cli, core
... forms, language-service, material, platform-browser
... platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1201.2
@angular-devkit/build-angular   12.1.2
@angular-devkit/core            12.1.2
@angular-devkit/schematics      12.1.2
@schematics/angular             12.1.2
rxjs                            6.6.7
typescript                      4.3.5
```

**Comment by JoostK** [comment]
Could you please share a minimum reproduction? The provided info is not quite sufficient enough to tell what is going on here.

**Comment by infacto** [comment]
Ok, my mistake. Sorry. I forgot to add `animations: [tabAnimation]` in the `Component` decorator. So I have to interpret the error message:

> Found the synthetic property @tabAnimation. Please include either "BrowserAnimationsModule" or "NoopAnimationsModule" in your application.

in: 

> Property @tabAnimation not found.

or: 

> Animation for @tabAnimation not found.

You guess it, we might change / detect this case for a better / more precise error message.

**Comment by JoostK** [comment]
That was addressed in #41356 though (12.2 it appears), so please consider upgrading to a more recent version.
